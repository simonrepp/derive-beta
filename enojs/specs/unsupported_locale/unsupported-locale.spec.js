const { parse } = require('../../eno.js');

describe('Unsupported locale', () => {
  test('throws', () => {
    expect(() => parse('', 'abcdef')).toThrowErrorMatchingSnapshot();
  });
});
