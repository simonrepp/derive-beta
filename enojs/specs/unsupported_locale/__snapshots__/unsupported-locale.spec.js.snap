// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unsupported locale throws 1`] = `"The requested locale \\"abcdef\\" is not supported. Translation contributions are very welcome and an easy thing to do - only a few easy messages need to be translated!"`;
